crates/astria-composer/src/searcher/mod.rs

```diff
@@ -25,25 +25,24 @@ use tracing::{
 
 use crate::Config;
 
-mod collector;
 mod executor;
+mod geth_collector;
 mod rollup;
-
-use collector::Collector;
 use executor::Executor;
+use geth_collector::GethCollector;
 
 /// A Searcher collates transactions from multiple rollups and bundles them into
 /// Astria sequencer transactions that are then passed on to the
 /// Shared Sequencer. The rollup transactions that make up these sequencer transactions
-/// have have the property of atomic inclusion, i.e. if they are submitted to the
+/// have the property of atomic inclusion, i.e. if they are submitted to the
 /// sequencer, all of them are going to be executed in the same Astria block.
 pub(super) struct Searcher {
     // Channel to report the internal status of the searcher to other parts of the system.
     status: watch::Sender<Status>,
     // The collection of collectors and their rollup names.
-    collectors: HashMap<String, Collector>,
+    geth_collectors: HashMap<String, GethCollector>,
     // The collection of the collector statuses.
-    collector_statuses: HashMap<String, watch::Receiver<collector::Status>>,
+    collector_statuses: HashMap<String, watch::Receiver<geth_collector::Status>>,
```

Comment:
`geth_collector_statuses` maybe?

Reply:
yeah that would be better. I initially thought we would have multiple collectors.